### ISSUE TITLE
add easylist polish to default set for polish

### ISSUE THREAD

**Issue by KonoromiHimaries** [NanoAdblocker/NanoCore#229]
### Describe the issue (Required)

https://easylist-downloads.adblockplus.org/easylistpolish.txt

> adguard, adblock plus, and adblock already has this list in the default set

**Comment by jspenguin2017** [comment]
What's wrong with the current one?

**Comment by KonoromiHimaries** [comment]
does not support internal advertising

**Comment by jspenguin2017** [comment]
What is "internal advertising"? Can you elaborate?

**Comment by KonoromiHimaries** [comment]
https://github.com/MajkiIT/polish-ads-filter/issues/9026

> <https://github.com/MajkiIT/polish-ads-filter/issues/8921>, <https://github.com/MajkiIT/polish-ads-filter/issues/8928>, <https://github.com/MajkiIT/polish-ads-filter/issues/8931>, <https://github.com/MajkiIT/polish-ads-filter/issues/8937>, <https://github.com/MajkiIT/polish-ads-filter/issues/8938>, <https://github.com/MajkiIT/polish-ads-filter/issues/8940>, <https://github.com/MajkiIT/polish-ads-filter/issues/8943>, <https://github.com/MajkiIT/polish-ads-filter/issues/8947>, <https://github.com/MajkiIT/polish-ads-filter/issues/8988>, <https://github.com/MajkiIT/polish-ads-filter/issues/9005>, <https://github.com/MajkiIT/polish-ads-filter/issues/9008>, <https://github.com/MajkiIT/polish-ads-filter/issues/9012>, <https://github.com/MajkiIT/polish-ads-filter/issues/9023>

**Comment by jspenguin2017** [comment]
I subscribed to EasyList Polish and I see no change in the links you provided.

**Comment by KonoromiHimaries** [comment]
@jspenguin2017 before you must report all internal ads here: https://github.com/easylistpolish/easylistpolish/issues

MajkiIT doesn't support internal ads

**Comment by jspenguin2017** [comment]
I still don't know what are "internal ads", can you elaborate better? 

Also, how do you know EasyList Polish will "support internal ads"?

**Comment by KonoromiHimaries** [comment]
@gargaamel, @kulfoon, @krystian3w

**Comment by kulfoon** [comment]
- internal ads are ads related to the website, promoting the website (self-promotion) and linking to the website's internal subdomains, they still are just ads like all other ads. 
- another kind of ads which are not always blocked by current POL list are leftovers and subtitles of ads like "Reclama", the ad is removed but the annoying subtitle still not, you can see such subtitles on the above screenshots

**Comment by kulfoon** [comment]
And what about the ads subtitles and remains.

**Comment by jspenguin2017** [comment]
Also, as far as I understood, many ads placeholders on Polish website are difficult to remove, and I highly doubt EasyList Polish can handle them.

**Comment by jspenguin2017** [comment]
I will reconsider this when there is a clear reason why EasyList Polish is a good complement to the current Polish filter. 
But as of now, it's pure speculation. Please, report issues to EasyList Polish, and we can revisit this later.

**Comment by kulfoon** [comment]
> jspenguin2017 : Also, as far as I understood, many ads placeholders on Polish website are difficult to remove, and I highly doubt EasyList Polish can handle them.

Yes, many of them are hard to remove, and we can live with them unfixed, but many of them are easy to remove...I can provide a bunch of example issues with annoying "reclama" text all over the website and which have been closed and left unfixed:
https://github.com/MajkiIT/polish-ads-filter/issues/9154
https://github.com/MajkiIT/polish-ads-filter/issues/9156
https://github.com/MajkiIT/polish-ads-filter/issues/9150
https://github.com/MajkiIT/polish-ads-filter/issues/9152
https://github.com/MajkiIT/polish-ads-filter/issues/9151
https://github.com/MajkiIT/polish-ads-filter/issues/9144
https://github.com/MajkiIT/polish-ads-filter/issues/9142
https://github.com/MajkiIT/polish-ads-filter/issues/9143
https://github.com/MajkiIT/polish-ads-filter/issues/9117
https://github.com/MajkiIT/polish-ads-filter/issues/9122
https://github.com/MajkiIT/polish-ads-filter/issues/9139
https://github.com/MajkiIT/polish-ads-filter/issues/9135
https://github.com/MajkiIT/polish-ads-filter/issues/9131
https://github.com/MajkiIT/polish-ads-filter/issues/9129
https://github.com/MajkiIT/polish-ads-filter/issues/9211
https://github.com/MajkiIT/polish-ads-filter/issues/9127

I don't know if they are hard to fix (didn't check that), but can check yourself if they can be easy removed or not.

**Comment by jspenguin2017** [comment]
If you think they can be easily removed, submit a Pull Request to the current Polish filters repository. If they don't accept it, submit to uAssets.

**Comment by kulfoon** [comment]
I was doing already PRs for such kind of issues, but the admin suddenly stopped to accept my PR anymore without any kind of explanation: 
`https://github.com/MajkiIT/polish-ads-filter/pulls?q=is%3Apr+is%3Aclosed+author%3Akulfoon`

Just take a look at my last PR
 `https://github.com/MajkiIT/polish-ads-filter/pull/9091/files#diff-711027ed0794992ae3b8710d874f3df7R2794`,
 it was fixing my typo from the previous PR, but still not fixed and POL list  contains bugged (for ABP) filtes for `www.bolec.info`.

**Comment by jspenguin2017** [comment]
Your PR contains way more than just a typo fix. Also, whether it is bugged for ABP is outside of my concern. 

Either way, that is not an argument for adding EasyList Polish to the resource manifest. 

Please, report issues that the regional filter won't fix to uAssets.

**Comment by kulfoon** [comment]
Whatever, I'm out of the thread.

**Comment by kulfoon** [comment]
That's great, after a month, during this month I've been checking several times every several days and it wasn't fixed, glad to see that it has been fixed at last.

**Comment by jspenguin2017** [comment]
I have no control over what uAssets accepts. If they don't want to fix it, report it to EasyList Polish, if a significant amount are fixed there, we can reopen this issue.

**Comment by kulfoon** [comment]
They won't accept it, KonoromiHimaries we must try to report to EasyList Polish.

**Comment by jspenguin2017** [comment]
Well, that is disappointing. 

Please report them to EasyList Polish. If they don't accept them, come back to me, I will have a solution for you. 

Let's get into action!

**Comment by kulfoon** [comment]
Ok, done: https://github.com/easylistpolish/easylistpolish/issues

**Comment by kulfoon** [comment]
Resolution fix issue as per https://github.com/easylistpolish/easylistpolish/issues/20#issuecomment-439739147: 30 hours

30 hours vs 56 days, well I'm positively surprised by such difference.

**Comment by krystian3w** [comment]
It is a pity that they also do not block self promotion (internal advertising) and do not fight with anti-adblock messeges / walls (there are legal limitations / fear of lawsuits against Eyeo GmbH).

**Comment by kulfoon** [comment]
Yea, but better stop complaining as per https://github.com/NanoAdblocker/NanoCore/issues/229#issuecomment-439146794:

> jspenguin2017 : I have no control over how ~~gorhill~~ EasyList Polish manages his projects. Please, do not bring your complains here.

**Comment by jspenguin2017** [comment]
That's a good start. Keep it up! 

> seems to be speculations only

If you have a good solution, why not consider submitting a pull request to EasyList Polish?

**Comment by jspenguin2017** [comment]
Why don't you benchmark your solutions and show him some hard data?

**Comment by kulfoon** [comment]
Because I want him to do so first ==> https://github.com/easylistpolish/easylistpolish/issues/20#issuecomment-440337970, coz he was the first who had mentioned about the performance impact.

**Comment by kulfoon** [comment]
Well, as for now it's hard to say/determine yet.

As per the first set of websites reported to EasyList Polish repo (16 websites): 
https://github.com/easylistpolish/easylistpolish/issues?q=is%3Aissue+is%3Aclosed+author%3Akulfoon
several have been fixed in EasyList Polish, 
several have been fixed in POL: polskie filtry do Adblocka i uBlocka 
and ​only 2 webpages haven't been fixed in any of the lists:
https://github.com/easylistpolish/easylistpolish/issues/11
https://github.com/easylistpolish/easylistpolish/issues/13

Now there is the second set of websites ongoing:
https://github.com/easylistpolish/easylistpolish/issues/created_by/kulfoon

So we wait to see how many will remain unfixed this time.

**Comment by jspenguin2017** [comment]
I'm talking about creating a filter list specifically for dealing with leftover ads placeholders. For all languages. 

Unless EasyList Polish is willing to remove every leftover placeholder on Polish website, I'm probably not going to add the filter list to the assets manifest. 

The filter list is pretty long, so I really don't want to add a regional filter for a few dozen cases. Beside, I have a bunch of ads placeholder issues on the backlog: https://github.com/NanoMeow/QuickReports/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22s%3A+backlog%22+label%3A%22t%3A+ads%22

**Comment by kulfoon** [comment]
> jspenguin2017 : Unless EasyList Polish is willing to remove every leftover placeholder on Polish website, I'm probably not going to add the filter list to the assets manifest.

Well, that will happen only if the admin of EasyList Polish will decide to create a small supplement list in his repo as previoulsy discussed here https://github.com/easylistpolish/easylistpolish/issues/20: It will be more clear whether such list will be created or not, after the second set of websites will be processed.

>   jspenguin2017 : I'm talking about creating a filter list specifically for dealing with leftover ads placeholders and similar minor issues. **For all languages**.

Oh. I have a question, would be the filter list as a single multi-language standalone list file or can it consists of separate partiucular lang sublist files included with include directive (so that everyone can pick his lang only)?

**Comment by kulfoon** [comment]
> jspenguin2017 : Unless EasyList Polish is willing to remove every leftover placeholder on Polish website, I'm probably not going to add the filter list to the assets manifest.

Well, that will happen only if the admin of EasyList Polish will decide to create a small supplement list in his repo as previoulsy discussed here https://github.com/easylistpolish/easylistpolish/issues/20: It will be more clear whether such list will be created or not, after the second set of websites will be processed.

>   jspenguin2017 : I'm talking about creating a filter list specifically for dealing with leftover ads placeholders and similar minor issues. **For all languages**.

Oh. Good idea.

**Comment by jspenguin2017** [comment]
Unless he is going to use `include` statement to automatically load the supplement list, I'm not going to add 2 filters just to remove some placeholders.

**Comment by jspenguin2017** [comment]
> Which phrase in english is a proper one related to such kind of ads?

I'm not native English speaker neither. 

None of them sound right, it's either "first-party ads" or "self promotion".

**Comment by kulfoon** [comment]
> jspenguin2017 : "first-party ads"

I think this is it. @KonoromiHimaries FYI.

**Comment by kulfoon** [comment]
OK, the second set of websites has been processed today, from 13 websites:

- 2 websites: (https://github.com/easylistpolish/easylistpolish/issues/25 and https://github.com/easylistpolish/easylistpolish/issues/28) turned out to be affected by other filter lists, so the second set had 11 websites
- 6 websites have been declined to be fixed as "not worth to fix":
https://github.com/easylistpolish/easylistpolish/issues/29
https://github.com/easylistpolish/easylistpolish/issues/32
https://github.com/easylistpolish/easylistpolish/issues/33
https://github.com/easylistpolish/easylistpolish/issues/34
https://github.com/easylistpolish/easylistpolish/issues/35
https://github.com/easylistpolish/easylistpolish/issues/36
- 1 website:
https://github.com/easylistpolish/easylistpolish/issues/37
has been declined to be fixed as per 
https://github.com/easylistpolish/easylistpolish/issues/37#issuecomment-449416387
- 3 websites have been bad fixed - the issue hasn't been fixed:
https://github.com/easylistpolish/easylistpolish/issues/26
https://github.com/easylistpolish/easylistpolish/issues/27
https://github.com/easylistpolish/easylistpolish/issues/31
- 1 website has been fixed partially:
https://github.com/easylistpolish/easylistpolish/issues/30

**Comment by jspenguin2017** [comment]
Alright, I created new filters for these placeholders. 
https://github.com/NanoAdblockerLab/NanoContrib

Please open pull requests and port all necessary filters over.

**Comment by jspenguin2017** [comment]
The new filter list will become active on the next release of Nano Adblocker. It will not be selected by default and will not be privileged.

**Comment by kulfoon** [comment]
The third set of websites has been processed today, from 13 websites:
-  1 websites https://github.com/easylistpolish/easylistpolish/issues/42 turned out to be a self-promotion, so the third set had 12 websites
- 4 websites have been fixed
https://github.com/easylistpolish/easylistpolish/issues/40
https://github.com/easylistpolish/easylistpolish/issues/43
https://github.com/easylistpolish/easylistpolish/issues/44
https://github.com/easylistpolish/easylistpolish/issues/48
- 1 website https://github.com/easylistpolish/easylistpolish/issues/49 has been declined to be fixed as "Anti-adblock wall"
- 1 website https://github.com/easylistpolish/easylistpolish/issues/47 has been fixed partially, the "reklama" 1 still exists
- 1 website https://github.com/easylistpolish/easylistpolish/issues/46 has been badly fixed, the issue still persist
- 4 websites have been declined to be fixed as "Not worth to add" 
https://github.com/easylistpolish/easylistpolish/issues/38
https://github.com/easylistpolish/easylistpolish/issues/39
https://github.com/easylistpolish/easylistpolish/issues/41
https://github.com/easylistpolish/easylistpolish/issues/45

**Comment by kulfoon** [comment]
The fourth set of websites has been processed recently, from 39 reported websites:

- 1 website the problem couldn't be reproduced anymore neither by the author nor by me, probably some 3rd party list fixed it, or the ad placeholder dissappeared:
https://github.com/easylistpolish/easylistpolish/issues/72
- 10 websites have been fixed completely:
https://github.com/easylistpolish/easylistpolish/issues/52
https://github.com/easylistpolish/easylistpolish/issues/56
https://github.com/easylistpolish/easylistpolish/issues/58
https://github.com/easylistpolish/easylistpolish/issues/60
https://github.com/easylistpolish/easylistpolish/issues/64
https://github.com/easylistpolish/easylistpolish/issues/75
https://github.com/easylistpolish/easylistpolish/issues/77
https://github.com/easylistpolish/easylistpolish/issues/80
https://github.com/easylistpolish/easylistpolish/issues/84
- 4 websites have been fixed partially, the unfixed part declined as "Not worth to add":
https://github.com/easylistpolish/easylistpolish/issues/53
https://github.com/easylistpolish/easylistpolish/issues/55
https://github.com/easylistpolish/easylistpolish/issues/68
https://github.com/easylistpolish/easylistpolish/issues/79
- 24 websites have been declined to be fixed, most of them as "Not worth to add", otherwise the reason is provided in the brackets:
https://github.com/easylistpolish/easylistpolish/issues/54
https://github.com/easylistpolish/easylistpolish/issues/57
https://github.com/easylistpolish/easylistpolish/issues/59
https://github.com/easylistpolish/easylistpolish/issues/61
https://github.com/easylistpolish/easylistpolish/issues/62 (as "This is link to advertising offer")
https://github.com/easylistpolish/easylistpolish/issues/63
https://github.com/easylistpolish/easylistpolish/issues/65
https://github.com/easylistpolish/easylistpolish/issues/66
https://github.com/easylistpolish/easylistpolish/issues/67
https://github.com/easylistpolish/easylistpolish/issues/69
https://github.com/easylistpolish/easylistpolish/issues/70
https://github.com/easylistpolish/easylistpolish/issues/71
https://github.com/easylistpolish/easylistpolish/issues/73
https://github.com/easylistpolish/easylistpolish/issues/74
https://github.com/easylistpolish/easylistpolish/issues/76
https://github.com/easylistpolish/easylistpolish/issues/78
https://github.com/easylistpolish/easylistpolish/issues/81
https://github.com/easylistpolish/easylistpolish/issues/82
https://github.com/easylistpolish/easylistpolish/issues/83 
https://github.com/easylistpolish/easylistpolish/issues/85 (as "Page uses anti-adblock. I won't block ads on these pages", however anyway it seems that the reported ads have been removed by some 3rd party lists or just disappeared)
https://github.com/easylistpolish/easylistpolish/issues/86 (as "Page is inactive since 2015")
https://github.com/easylistpolish/easylistpolish/issues/87
https://github.com/easylistpolish/easylistpolish/issues/88
https://github.com/easylistpolish/easylistpolish/issues/89
https://github.com/easylistpolish/easylistpolish/issues/90

**Comment by jspenguin2017** [comment]
As I said, I will not add EasyList Polish because they are declining too many issues. 

Please send Pull Requests to the contrib filter here: https://github.com/NanoAdblockerLab/NanoContrib

Locking.
